client/messaging/chat.tsx

```diff
@@ -38,12 +38,8 @@ export default function Chat(props: ChatProps) {
       if (newIsScrolledUp !== isScrolledUp) {
         setIsScrolledUp(newIsScrolledUp)
       }
-
-      if (props.listProps.onScrollUpdate) {
-        props.listProps.onScrollUpdate(target)
-      }
     },
-    [isScrolledUp, props.listProps.onScrollUpdate],
+    [isScrolledUp],
```

Comment:
Doesn't need to be done in this PR, but I think we could use the same IntersectionObserver to update this state (have an Observer at the bottom, when it's not intersecting `isScrolledUp = true`)

Seems like a potentially common case for users of infinite-scroll-list